web/components/TextBlock/Speakers/Speakers.tsx

```diff
@@ -1,25 +1,75 @@
-import { Person } from '../../../types/Person';
-import { PortableTextComponentProps } from '@portabletext/react/dist/react-portable-text.esm';
-import { EntitySectionSelection } from '../../../types/EntitySectionSelection';
+import type { PortableTextComponentProps } from '@portabletext/react';
+import Link from 'next/link';
+import { imageUrlFor } from '../../../lib/sanity';
+import type { EntitySectionSelection } from '../../../types/EntitySectionSelection';
+import type { Person } from '../../../types/Person';
+import type { SimpleCallToAction } from '../../../types/SimpleCallToAction';
 import { getCollectionForSelectionType } from '../../../util/entity';
+import { getEntityPath } from '../../../util/entityPaths';
+import ButtonLink from '../../ButtonLink';
+import GridWrapper from '../../GridWrapper';
+import styles from './Speakers.module.css';
 
 type SpeakersProps = {
   type: EntitySectionSelection;
+  heading?: string;
+  callToAction?: SimpleCallToAction;
   allSpeakers?: Person[];
   speakers?: Person[];
 };
 
 export const Speakers = ({
-  value: { type, allSpeakers, speakers },
+  value: { type, heading, callToAction, allSpeakers, speakers },
 }: PortableTextComponentProps<SpeakersProps>) => (
-  <>
-    {getCollectionForSelectionType(type, allSpeakers, speakers).map(
-      (speaker) => (
-        <div key={speaker._id}>
-          <div>{speaker.name}</div>
-          <div>{speaker.title}</div>
-        </div>
-      )
-    )}
-  </>
+  <section className={styles.container}>
+    <GridWrapper>
+      <div className={styles.introContent}>
+        {heading && <h2 className={styles.heading}>{heading}</h2>}
+        {callToAction && (
+          <ButtonLink
+            url={
+              callToAction.link?.external ||
+              callToAction.link?.internal?.slug?.current
+            }
+            text={callToAction.text}
+          />
```

Comment:
This seems to give a warning locally:

`Prop `href` did not match. Server: "program" Client: "/program"`

Can it use `web/components/SimpleCallToAction` instead?

Reply:
Excellent point! e88f9f7